Adds instructions for how to resolve "Cannot find module `./<module n…

SanjoSolutions opened the pull request:
Adds instructions for how to resolve "Cannot find module `./<module name>.js` from `./<module_name>.spec.ts`".

Biki-das reviewed (commented):
thanks @SanjoSolutions , can you apply these changes to other docs as well including the `docs` folder too.

SanjoSolutions commented:
@Biki-das Thanks for the review. I have added the instructions also to the docs for the other versions of Jest. And I have also added the instructions to the `GettingStarted.md` in the `docs` folder.

Biki-das commented:
> Author

@SanjoSolutions LGTM!
